### PR TITLE
Added eXplosive Dinosaurs demo package

### PR DESCRIPTION
**Pull request by panreyes**
This package includes PixTudio ( pixtudio.org ), a 2D open source game engine (zlib license), and eXplosive Dinosaurs' demo, a closed source party game I created. It has been tested in a WebOS 3.4 LG TV from 2017 (49UH661V).

### CODE REVIEW

**Comment by mariotaku** [comment]
Thank you for your contribution! Here are few reviews for this app:

### Issues
*  `pmouya` is not executable, after changing to 0755, the app launches without issue

### Suggestions
* App is too large. On many TVs, only < 500MB storage is available. Wonder if it's possible to separate game data and ask user to download (preferably to external storage if present)
* Support magic remote as an input source maybe?

**Comment by mariotaku** [comment]
Also we need to discuss about licensing issue, as we really want to enrich our repo with a variety of games/apps, we're thinking to host open source apps only. Maybe this should be changed, or done in another way.

**Comment by Informatic** [comment]
Hey! Great to see you managed to build the engine and it's feasible to bundle it in an ipk. :D

> `pmouya` is not executable, after changing to 0755, the app launches without issue

I am assuming @panreyes didn't notice that because devmode init script makes whole `/media/developer` `0777` recursively on startup, and appInstallService retains existing file permissions on reinstalls/updates.

Regarding hosting of non-free apps:
One thing we could do is to have a separate repository like Debian/Ubuntu `non-free`/`multiverse`, still hosted in here, just exported on a different URL, and an option in Homebrew Channel settings to enable that. We already have infrastructure for that in Homebrew Channel and adding an actual switch for adding/removing an extra optional default repo would be <1h of work here. I would then opt for adding a clear description that this sub-repo contains non-free software and we can't really vet on it in any way. (outside of basic testing)

**Comment by mariotaku** [comment]
@Informatic does `https://repo.webosbrew.org/api/non-free/` endpoint sound good to you?

**Comment by mariotaku** [comment]
Or I can create another repo/domain for that.

File: packages/com.rawrlab.pmouya.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+title: eXplosive Dinosaurs (Demo)
+iconUri: https://github.com/panreyes/xd-webos-ipk/releases/download/v1.0.0/icon160.png
+manifestUrl: https://github.com/panreyes/xd-webos-ipk/releases/download/v1.0.0/com.rawrlab.pmouya.manifest.json
+category: games
+description: |
```
Comment (mariotaku) [inline packages/com.rawrlab.pmouya.yml:5]: This is a very nice game, please consider to add more screenshots, and information in the description field. [Example](https://repo.webosbrew.org/apps/com.limelight.webos)

File: packages/com.rawrlab.pmouya.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+title: eXplosive Dinosaurs (Demo)
+iconUri: https://github.com/panreyes/xd-webos-ipk/releases/download/v1.0.0/icon160.png
+manifestUrl: https://github.com/panreyes/xd-webos-ipk/releases/download/v1.0.0/com.rawrlab.pmouya.manifest.json
+category: games
```
Comment (mariotaku) [inline packages/com.rawrlab.pmouya.yml:4]: Please add `pool: non-free`.

**Review by mariotaku — CHANGES_REQUESTED** [review]

**Review by mariotaku — APPROVED** [review]

**Comment by mariotaku** [comment]
All good now. I have more suggestions for later. The binary and picture assets can be hosted on website of this game IMO, since it's not required to be open source, you don't have to use Github to host binaries.

**Pull request merged**